recipes/mvapich2/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+numpy:
+  - 1.22  # Specifying a version to eliminate the warning
```

Comment:
This should probably be removed to follow what's used elsewhere in conda-forge

Reply:
OK